### DISCUSSION TITLE
Welcome to Rescue Discussions!

### DISCUSSION THREAD

**Discussion by swapnilsparsh** [swapnilsparsh/Rescue#145] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by swapnilsparsh** [comment]
For people looking to open issues, here's a simple flow 👍🏼

1. Understand the project. Find the gaps or issues there.
2. Look into the existing issues, if they have been raised and see if you can solve them.
3. If you think you can raise a new issue, go for it.
4. Claim an issue and get it assigned. Then start working.
5. You can work by forking the project.
6. Create a new branch. Change the files.
7. Git Add, Commit, Push to your branch of your repo.
8. Create a PR. Wait till they review and get it merged.
9. Any changes, push commits to same branch.
10. Happy Coding 👩🏼‍💻

**Comment by tushartiwari7** [comment]
Is this project a part of any Open Source Community.

**Reply by swapnilsparsh** [reply]
No, but as of now its there under GirlScript Summer of Code 2021 event.

**Reply by tushartiwari7** [reply]
**Cool**! I would surely like to contribute in this
